sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters/src/autorest.md

```diff
@@ -26,8 +24,6 @@ sample-gen:
 skip-csproj: true
 modelerfour:
   flatten-payloads: false
-use-model-reader-writer: true
```

Comment:
Why do we delete those two lines? That causes a series of code changes.

Reply:
I put these back and regenerated the sdk.